api/v1alpha1/drclusterpeers_types.go

```diff
@@ -34,44 +34,45 @@ const (
 	Async ReplicationPolicyType = "async"
 )
 
-// ClusterPeersSpec defines the desired state of ClusterPeers
-type ClusterPeersSpec struct {
+// DRClusterPeersSpec defines the desired state of DRClusterPeers
+type DRClusterPeersSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// Members of the ClusterPeers set
-	ClusterIDs []string `json:"clusterIDs"`
+	// Members of the DRClusterPeers set
+	ClusterNames []string `json:"clusterNames"`
```

Comment:
(just leaving a note based on older discussions): We may NOT want ManagedClusterSet (MCS) as the field here yet, as ManagedClusterSet enforces a cluster to be in only one set and we may want different intersecting sets. OTOH, MCS CRD and its reconciler provides any checks that ensures clusters in MCS are valid, which we are missing at present.